regolith/helpers/u_milestonehelper.py

```diff
@@ -161,7 +161,7 @@ def db_updater(self):
             doc = all_milestones[rc.index-2]
             identifier = doc['identifier']
             if not doc.get('type') and not rc.type and identifier=='milestones':
-                    raise RuntimeError("This milestone does not has 'type' specified in the database. "
+                    raise RuntimeError("Missing 'type' in the database for this milestone. "
```

Comment:
I was thinking more like
"ERROR: This milestone does not have a type set and this is required.  Please rerun your command adding "--type" and typing a type from this list: {ALLOWED_TYPES}"